src/platform/Tizen/SystemInfo.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "SystemInfo.h"
+
+#include <iostream>
+
+#include <system_info.h>
+#include <tizen.h>
+
+#include <lib/support/logging/CHIPLogging.h>
+
+using namespace std;
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+SystemInfo SystemInfo::sInstance;
+
+double SystemInfo::GetPlatformVersion()
```

Comment:
How about using `unsigned int` for version components instead of double. E.g.:
```c++
struct PlatformVersion {
  uint8_t mMajor;
  uint8_t mMinor;
};

CHIP_ERROR SystemInfo::GetPlatformVersion(PlatformVersion & version) {
...
}
```

Reply:
It seems great. Thank you.